modules/security-storage-persistent.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-storage.adoc
+
+[id="security-network-storage-persistent_{context}"]
+=  Persistent volume plug-ins
+
+Containers are useful for both stateless and stateful applications.
+Protecting attached storage is a key element of securing stateful services.
+Using the Container Storage Interface (CSI), {product-title} can
+incorporate storage from any storage back end that supports the CSI interface.
+Protecting attached storage is a key element of securing stateful services.
+
+{product-title} provides plug-ins for multiple types of storage, including:
+
+* AWS Elastic Block Stores (EBS) *
+* AWS Elastic File System (EFS) *
+* GCE Persistent Disks *
+* Azure Disk *
+* Azure File *
+* OpenStack Cinder *
+* VMware vSphere * 
+* Network File System (NFS)
+* Fibre Channel
+* GlusterFS
+* iSCSI
+* RADOS (Ceph)
```

Comment:
Good catch. I'll update the list.

Reply:
I'm going to leave off local volumes and hostPath, since they are not supported in production. Otherwise, I updated the list based on the storage we documented for 4.4.